Framework/DataHandling/src/LoadLogsForSNSPulsedMagnet.cpp

```diff
@@ -194,12 +194,13 @@ struct Pulse {
   /// less than one second.
   uint32_t nanoseconds;
 
-  /// The number of seconds since January 1, 1990.
-  uint32_t seconds;
-
   /// The index of the first event for this pulse.
+  // cppcheck-suppress unusedStructMember
   uint64_t event_index;
 
+  /// The number of seconds since January 1, 1990.
+  uint32_t seconds;
+
```

Comment:
I think you accidentally swapped the order when adding this back in after the initial changes. As a consequence the struct is now bigger than it was before (since `uint64_t` has a higher alignment requirement).